Lib/test/test_zlib.py

```diff
@@ -439,7 +439,14 @@ def test_flushes(self):
         sync_opt = ['Z_NO_FLUSH', 'Z_SYNC_FLUSH', 'Z_FULL_FLUSH',
                     'Z_PARTIAL_FLUSH']
 
-        ver = tuple(int(v) for v in zlib.ZLIB_RUNTIME_VERSION.split('.'))
+        v = zlib.ZLIB_RUNTIME_VERSION.split('-', 1)[0].split('.')
+        if len(v) < 4:
+            v.append('0')
+        elif not v[-1].isnumeric():
+            v[-1] = '0'
+
```

Comment:
These 5 lines appear also in test_wbits, wouldn't it be better to extract them into a helper function?

Reply:
maybe, i'll have a look if i or someone make an android buildbot someday and/or have some time left and testsuite can actually run.